packages/plugins/typescript/operations/src/visitor.ts

```diff
@@ -205,15 +207,42 @@ export class TypeScriptDocumentsVisitor extends BaseDocumentsVisitor<
       : [];
   }
 
+  public getExternalSchemaTypeImports(): Array<string> {
+    if (!this.config.importSchemaTypesFrom) {
+      return [];
+    }
+
+    const hasTypesToImport = Object.keys(this._usedNamedInputTypes).length > 0;
+
+    if (!hasTypesToImport) {
+      return [];
+    }
+
+    return [
+      generateImportStatement({
+        baseDir: '',
+        baseOutputDir: '',
+        outputPath: '',
+        importSource: {
+          path: '',
+          namespace: this.config.namespacedImportName,
+          identifiers: [],
+        },
```

Comment:
TODO: handle pathing to schema type file (also generated by `typescript-operations`)